backend/services/implementations/__tests__/testSessionService.test.ts

```diff
@@ -42,4 +42,27 @@ describe("mongo testSessionService", (): void => {
     assertResponseMatchesExpected(mockTestSession, res[0]);
     assertResultsResponseMatchesExpected(mockTestSession, res[0]);
   });
+
+  it("getTestSessionsByTestId", async () => {
+    await MgTestSession.create(mockTestSessionsWithSameTestId);
+    const testId = "62c248c0f79d6c3c9ebbea95";
+
+    const res = await testSessionService.getTestSessionsByTestId(testId);
+    expect(res.length).toEqual(mockTestSessionsWithSameTestId.length);
+    for (let i = 0; i < res.length; i += 1) {
+      assertResponseMatchesExpected(mockTestSessionsWithSameTestId[i], res[i]);
+      assertResultsResponseMatchesExpected(
+        mockTestSessionsWithSameTestId[i],
+        res[i],
+      );
+    }
+  });
+
+  it("getTestSessionsByTestId for non-existing id", async () => {
+    await MgTestSession.insertMany(mockTestSession);
```

Comment:
```suggestion
    await MgTestSession.create(mockTestSession);
```
should be this if i'm understanding correctly?